src/Administration/Resources/app/administration/build/plugins.vite.ts

```diff
@@ -29,11 +30,32 @@ import { loadExtensions, findAvailablePorts } from './vite-plugins/utils';
 import type { ExtensionDefinition } from './vite-plugins/utils';
 import injectHtml from './vite-plugins/inject-html';
 
+/**
+ * This function returns the IP address of the container
+ * if the application is running in a Docker container.
+ */
+const getContainerIP = (): string|undefined => {
+    const interfaces = os.networkInterfaces();
+
+    let containerIP;
+
+    Object.values(interfaces).forEach(ifaces => {
+        ifaces?.forEach(iface => {
+            if (!iface.internal && iface.family === 'IPv4') {
+                containerIP = iface.address;
+            }
+        });
+    });
+
+    return containerIP;
+};
```

Comment:
```suggestion
const getContainerIP = (): string | undefined => {
    const interfaces = os.networkInterfaces();

    for (const ifaces of Object.values(interfaces)) {
        for (const iface of ifaces || []) {
            if (!iface.internal && iface.family === 'IPv4') {
                return iface.address;
            }
        }
    }

    return undefined;
};
```

Reply:
And what about `IPv6`?

Reply:
Using a old `for` loop is prevented by our ESLint rules, see https://eslint.org/docs/latest/rules/no-restricted-syntax

For IPv6: isn't it just an additional IP to access it? The interface check looks for the IPv4 address which should still be available. Correct me if I am wrong but I don't use Docker personally.